### PR TITLE
Outifts tab navigation behaviour

### PR DESCRIPTION
**Pull request by changyyn**
This pull request involves many things.

- Fixes the bug where outfits tab is not shown as selected even when on one of the fragments of
- Implemented Praneer's earliar pull request, as it was integral and just easier to do here
- Shows OutfitHistory only if user has logged outfit for the day.
- Fixed a bug in OutfitsFragment.kt where invalid ClothingHistory would be inserted

### CODE REVIEW

**Comment by joshpeng1999** [comment]
Screen Shot 2022-12-04 at 9 36 14 PM
I'm able to log an outfit even when I don't have any clothes, is this expected behavior?

**Comment by joshpeng1999** [comment]
At line 169  in outfitsuggestionFragment.kt remove the commented print statement. Otherwise LGTM good jobbbb

File: app/src/main/java/com/example/capsule/ui/outfitSuggestion/OutfitSuggestionFragment.kt [hunk 1]
```diff
@@ -195,14 +213,24 @@ class OutfitSuggestionFragment: Fragment(), LocationListener {
...
                 outfitSuggestionViewModel.insert(entry)
+                clothingInserted = true
+            }
+            if (clothingInserted) {
+                findNavController().navigate(R.id.action_navigation_outfits_suggestion_to_navigation_outfits_history)
+                with(requireActivity().getPreferences(Context.MODE_PRIVATE).edit()) {
+                    putLong(getString(R.string.has_inserted_for_day_key), calendar.timeInMillis)
+                    apply()
+                }
+                Toast.makeText(requireActivity(), R.string.empty_outfit_log, Toast.LENGTH_SHORT).show()
```
Comment (gracelu0) [inline app/src/main/java/com/example/capsule/ui/outfitSuggestion/OutfitSuggestionFragment.kt:224]: The string should be `R.string.outfit_logged` for this case right?

Comment (changyyn) [reply app/src/main/java/com/example/capsule/ui/outfitSuggestion/OutfitSuggestionFragment.kt:224]: Oh that's right.

**Pull request merged**